src/jdk.hotspot.agent/linux/native/libsaproc/symtab.c

```diff
@@ -390,7 +389,7 @@ static struct symtab* build_symtab_internal(int fd, const char *filename, bool t
         goto bad;
       }
 
-      rslt = hcreate_r(htab_sz, symtab->hash_table);
+      hcreate_r(htab_sz, symtab->hash_table);
       // guarantee(rslt, "unexpected failure: hcreate_r");
```

Comment:
The commented out guarantee line references rslt. I'm not so sure why it was commented out, but it goes back to the initial load of the file 17 years ago. It looks like the correct thing to do if rslt is null is to "goto bad;" but that change is probably beyond the scope of this PR. Maybe file a new CR.

Reply:
Hi Chris , I created https://bugs.openjdk.org/browse/JDK-8341820 for the return value checking.

Reply:
Might be a good idea to have rslt commented out rather than removed outright if we don't want to forget about it. guarantee on a variable that doesn't exist will be rather confusing to anyone reading through the code

Reply:
> Might be a good idea to have rslt commented out 

Why not; Chris what do you say ?

Reply:
Yeah, I think it is good like this.